sound/soc/sof/sof-priv.h

```diff
@@ -387,6 +387,7 @@ struct snd_sof_dev {
 	u32 pcm_active;		/* Count of active PCM substream */
 	u32 lp_pcm_active;	/* Count of active low power PCM substream */
 	struct delayed_work d0i3_entry_work;
+	bool kwd_at_s0i3; /* KWD run at S0I3 suspended */
```

Comment:
no, this is too specific and shouldn't be in snd_sof_dev.
And it's not clear to me why this is needed at all.